pages.pl/linux/uuid.md

```diff
@@ -0,0 +1,33 @@
+# uuid
+
+> Twórz i dekoduj uniwersalne identyfikatory (UUID).
+> Zobacz też `uuidgen`.
+> Więcej informacji: <https://manned.org/uuid>.
+
+- Stwórz UUIDv1 (oparte o zegar systemowy i - jeśli dostępne - adres sprzętowy):
+
+`uuid`
+
+- Stwórz UUIDv4 (losowy):
+
+`uuid -v {{4}}`
+
+- Stwórz wiele UUIDv4 na raz:
+
+`uuid -v {{4}} -n {{ilość_uuid}}`
+
+- Stwórz UUIDv4 w konkretnym formacie:
```

Comment:
The following fix can be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/uuid.md) page is updated:

```suggestion
- Stwórz UUIDv4 w konkretnym [F]ormacie:
```

Code descriptions can be changed to provide mnemonics for short options.